config/testgrids/config.yaml

```diff
@@ -702,9 +700,6 @@ dashboards:
 - name: conformance-all
   # entries are named $PROVIDER, $KUBERNETES_RELEASE
   dashboard_tab:
-  - name: kind, v1.14 (dev, ARM64)
```

Comment:
this should be in multiple dashboards in this file, if we're moving it, we should move them all or at least matching pairs

Reply:
Can we cut-paste the conformance dashboards to a separate `kind-conformance` file, then?

Reply:
this is the part I don´t get from the email, is this file still valid?
if not, do I need to create all the folders for the other dashboards?

Reply:
I'm not fully up to date on the new world of managing testgrid tabs but as for kind:

Right now we ensure that the `conformance-all` entries generally match the other `conformance-foo` dashboards

The same kind tabs should be in both, whatever works best for managing that.

Reply:
> this is the part I don´t get from the email announcing the change, is this file still valid?
> if not, do I need to create all the folders for the other dashboards?

The file is currently valid as-is, and will be valid as any YAML file in this directory, split up however. We want to split up the testgrid tabs according to who knows about what, to make it all more manageable. (see #13541).

I don't know who owns the `conformance-*` dashboards. If it's an existing sig, I recommend a separate file in that folder. If it's someone else, then make a directory for them.

The conformance dashboards can all go in one file, unless that's too big, and then they can be divided up in multiple files.